src/hessian.F90

```diff
@@ -175,6 +176,13 @@ subroutine numhess( &
 !! ========================================================================
 !  Hessian part -----------------------------------------------------------
 
+   !analytical hessian calculation
+   if(mode_extrun .eq. p_ext_turbomole) then    
+           dipd=0.0_wp
+           freezeset%n=0
```

Comment:
Overwriting an input variable is generally a bad idea, if we can't support this runmode we should fail rather than doing something else here.